.devcontainer/docker-compose.yml

```diff
@@ -2,27 +2,28 @@ version: '3'
 
 services:
   dev:
-    build: 
+    build:
       context: ..
       dockerfile: .devcontainer/Dockerfile
     environment:
       SQLALCHEMY_DATABASE_URI: postgresql://postgres:chummy@db/notification_api
       SQLALCHEMY_DATABASE_TEST_URI: postgresql://postgres:chummy@db/test_notification_api
       REDIS_URL: redis://redis:6380
     volumes:
-      - ..:/workspace:cached   
+      - ..:/workspace:cached
+      - /Users/andrewleith/projects/notification-utils:/nutils:cached
```

Comment:
🙈

Reply:
Good catch - did not mean to push that :D Will revert..